llvm/include/llvm/Analysis/CheriBounds.h

```diff
@@ -33,7 +34,7 @@ class CheriNeedBoundsChecker {
                                  const APInt &CurrentGEPOffset,
                                  unsigned Depth) const;
 
-  const Instruction *I;
+  const Instruction *RootInst;
```

Comment:
We should probably stop making this const so UsesThatNeedBounds can contain non-const uses and avoid needing to const_cast when updating them, but that's a separate issue...